salt/modules/virt.py

```diff
@@ -2809,6 +3006,184 @@ def _get_with_unit(node):
             "set": lambda n, v: n.set("dev", v),
             "del": salt.utils.xmlutil.del_attribute("dev"),
         },
+        {
+            "path": "cpu",
+            "xpath": "vcpu",
+            "get": lambda n: int(n.text),
+            "set": _set_vcpu,
+        },
+        {"path": "cpu:maximum", "xpath": "vcpu", "get": lambda n: int(n.text)},
+        {
+            "path": "cpu:placement",
+            "xpath": "vcpu",
+            "get": lambda n: n.get("placement"),
+            "set": lambda n, v: n.set("placement", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("placement"),
+        },
```

Comment:
If converting to `auto-placement` then this should be changed too

Reply:
It seems like we can pass `placement:auto` to indicate auto placement

Reply:
Sure, but the idea here was to use a boolean rather than a string to avoid the user to look for the possible values in the doc.